pyproject.toml

```diff
@@ -18,7 +18,7 @@ include_trailing_comma = true
 
 
 [tool.poetry.dependencies]
-python = "^3.8, <3.11"
+python = "^3.8, <3.12"
```

Comment:
Should this really be `3.12` or `3.11`?

Reply:
this dockerfile is 3.11, so you mean <= 3.11?
I think this causes issues with subversions like 3.11.1 thats why they do <next python version
I had an issue with 3.12 where i did a specific subversion and they released a new subversion breaking the project